js/main.js

```diff
@@ -0,0 +1,100 @@
+function Register() {
+    this.domElement = null;
+    this.entries = $(".entries");
+    this.inputBox = $(".newEntry");
+    this.totalEl = $(".total");
+    this.total = 0.00;
+    this.form = $("form");
+    this.form.on("submit", this.updateRegister);
+
+
+    this.initialize = function(container) {
+      this.domElement = $('<div class="register"></div>');
+      container.append(this.domElement);
+  };
+
+    this.render = function() {
+
+      var templateString =
+      "<div class='ticket'>" +
+      "<h1>Thank You!</h1>" +
```

Comment:
suggest using indentation to mimic how the HTML would be indented (for readability)